README.md

```diff
@@ -92,43 +94,321 @@ It even works on MacOSX in addition to the OS scroll acceleration,
+```
+This will stop the daemon and uninstall it. Note that this will not remove
+the configuration file.
+
+### Getting the status/log of the daemon
+
+To get the status of the daemon, you can use the `--daemon-status` option.
+```bash
+scroll-accelerator --daemon-status
+```
+This will print the status of the systemd unit associated with the daemon.
+
+To get the log of the daemon, you can use the `--daemon-log` option.
+```bash
+scroll-accelerator --daemon-log
+```
+This will print the log of the daemon.
+
+Note that this will be fairly empty by default.
+To get more verbose output, you can use the `--daemon-verbosity` option
+when installing or restarting the daemon:
+```bash
+scroll-accelerator --restart-daemon --daemon-verbosity 1
+```
+Then, after scrolling a few times, you can use `--daemon-log` again to see
+the log.
+It should now show a more verbose log, including the scroll events and the
+acceleration.
+
+## Acceleration parameters
+
+The following acceleration parameters determine the behavior of the scroll
+accelerator
+and can be set using the command line options or the configuration file.
+
+* `exp`: the exponential factor. This controls how fast the scroll speed
+  increases.
+* `multiplier`: the scalar multiplier. This is multiplied by the scroll
+  speed.
+* `threshold`: the threshold. This controls the minimum scroll speed.
+
+The exact formula for computing the target scroll speed is:
+```python
+m = (user_scroll_speed - threshold) ** exp
+target_scroll_speed = user_scroll_speed + m * multiplier
+```
 
-If you found values that work for you, you can install
-the script as a systemd user unit (only on Linux):
+## Uninstallation
 
-    ./install-daemon.py
+To uninstall `scroll-accelerator`, first uninstall the daemon:
+```bash
+scroll-accelerator --uninstall-daemon
+```
+
+Then you can uninstall the application using whichever package manager was
+used to install it.
+For pipx:
+```bash
+pipx uninstall scroll-accelerator
+```
+
+For `pip`:
+```bash
+pip uninstall scroll-accelerator
+```
+
+Note that this will not remove the configuration file.
+To remove the configuration file, you can run:
+```bash
+rm -rf ~/.config/scroll-accelerator
+```
+
+## Development/Contributing
+
+We welcome any and all contributions!
+
+We ask that you follow the guidelines below in order to make it easier to
+review and merge your changes.
 
-This will create a configuration file in
-`~/.config/mouse-scroll-wheel-accelerator/config.py`.
-Enter your preferred values there.
+### Prerequisites
+
+#### Uninstall the package (if applicable)
+
+If you already installed the package using `pipx` or some other method,
+it is highly recommended to uninstall it first.
+This will ensure that you are not using a version of the package that is not
+under development.
+See the [Uninstallation](#uninstallation) section for instructions.
+
+#### Install `uv`
+
+We ask that you use `uv` to manage the project and its dependencies.
```

Comment:
I would not formulate it such that any developer is required to use uv. I would leave that open to the developer.

Reply:
Why is there no reason? I think uv is anyway useful, and probably the tool of choice for many people. And I thought (similar as the installation instructions), having such instructions here could be helpful as a starting point for developers who are not familiar with the process.

For users/developers who are already familiar, they would not really need any such instructions anyway. For experienced developers, other questions, like what code style to follow, what rules to follow, etc, are more relevant. For experienced users, just a hint that the package is on PyPI would be enough.